src/gsad_gmp.c

```diff
@@ -4387,7 +4428,7 @@ append_alert_condition_data (GString *xml, params_t *data,
  */
 static void
 append_alert_method_data (GString *xml, params_t *data, const char *method,
-                          params_t *report_formats)
+                          params_t *report_formats, params_t *report_configs)
```

Comment:
I think `report_configs` is not used in this function. If it is not used, you might want to update the calling functions `create_alert_gmp` and `save_alert_gmp`.